pipelines/calibrate.py

```diff
@@ -1,13 +1,16 @@
+# TODO DM-40388: Remove this file, once the new pipeline is vetted.
```

Comment:
This comment refers to an ongoing project to replace `CalibrateTask` with `CalibrateImageTask` in the `ApVerify` pipeline. Do we have a "new pipeline" in PP?

Reply:
Ugh, I guess we have to, since `ApPipe.yaml` imports the main one. Perhaps "Remove this file when `ApPipe` is migrated to `CalibrateImageTask`"?

Reply:
From reading the tickets it looks to me that eventually `CalibrateImageTask` will replace `CalibrateTask` everywhere, so when that happens I assume we'd want PP to follow too?  DM-40388 looks like the right ticket for that, but it can be a separate ticket too. Thanks for adding a comment in DM-40388.  

Tagging @isullivan  @ebellm

Reply:
We certainly want the PP payload in sync with `ap_pipe` and `ap_verify`.